### PR TITLE
Added XCode setup documentation to Toga Tutorial

### PR DESCRIPTION
**Pull request by CaseyFaist**
### Docs/Tutorial update

It's not necessary to sign up for the full developer license to get started with BeeWare iOS, but found some helpful tips to share for users new to Xcode.

This doc updates the Getting Started RST to include configuration and a bit of troubleshooting for getting XCode ready to test the HelloWorld project from the Toga tutorial both locally and on an iPhone  🎉 ❤️ 🎉

### CODE REVIEW

File: docs/tutorial/getting-started.rst [hunk 1]
```diff
@@ -112,6 +112,22 @@ iOS
   install the Xcode Command Line Tools. This can be done from the Preference
   panel within XCode itself.
 
+* For first time XCode users, make sure you've installed or updated to the latest XCode version.
+
+* You'll need to sign in to XCode with your Apple ID account - the same one you'd use for iTunes.
+
+* On Mohave, the easiest way to ensure that the Xcode Command Line Tools are
```
Comment (phildini) [inline docs/tutorial/getting-started.rst]: Mojave

File: docs/tutorial/getting-started.rst [hunk 2]
```diff
@@ -112,6 +112,22 @@ iOS
   install the Xcode Command Line Tools. This can be done from the Preference
   panel within XCode itself.
 
+* For first time XCode users, make sure you've installed or updated to the latest XCode version.
+
+* You'll need to sign in to XCode with your Apple ID account - the same one you'd use for iTunes.
```
Comment (phildini) [inline docs/tutorial/getting-started.rst]: This is needed for developing or for deploying?

File: docs/tutorial/getting-started.rst [hunk 3]
```diff
@@ -112,6 +112,22 @@ iOS
...
+
+* On Mohave, the easiest way to ensure that the Xcode Command Line Tools are
+  installed is to open a default project template in XCode and run for the newest version of iPhone.
```
Comment (phildini) [inline docs/tutorial/getting-started.rst]: Is this method preferable to `xcode-select --install` from the terminal? I think going through the Xcode UI we might have to explain more steps (what is a project? How does one run it?)

File: docs/tutorial/getting-started.rst [hunk 4]
```diff
@@ -112,6 +112,22 @@ iOS
...
+
+* You can create a free personal team by clicking on ``Preferences > Accounts`` and adding your Apple ID account.
+
+* When you're ready to run the project, open it in XCode and double-click on Hello World.
+  The run button is on the top left of the screen.
+
+* To install on an iPhone, plug the iPhone into the computer, unclick ``Automatically
```
Comment (phildini) [inline docs/tutorial/getting-started.rst]: This is great. 💯

File: docs/tutorial/getting-started.rst (hunk 1 above)
Comment (CaseyFaist) [reply docs/tutorial/getting-started.rst]: Thank you! Spelling :shakes-fist:

File: docs/tutorial/getting-started.rst (hunk 2 above)
Comment (CaseyFaist) [reply docs/tutorial/getting-started.rst]: This is needed for developing - it is a prereq to be able to run the simulation on your computer or test on your phone after booting up the HelloWorld app.

Should I clarify with "To be able to run the simulator," ?

Comment (phildini) [reply docs/tutorial/getting-started.rst]: Yes, if that's what you needed. To be clear, you needed a developer account to get the simulator up and running? Russell and I have never seen this, but our machines should be considered development waste dumps at this point and are so far from the normal beginner experience we can see Saturn.

If you had to do this, then definitely add the documentation.

**Pull request merged**